src/test/java/com/puppycrawl/tools/checkstyle/checks/header/RegexpHeaderCheckTest.java

```diff
@@ -380,4 +380,20 @@ public void testNoHeaderMissingErrorInCaseHeaderSizeEqualToFileSize() throws Exc
         verify(checkConfig, getPath("InputRegexpHeaderMulti52.java"), expected);
     }
 
+    @Test
+    public void testHeaderWithInvalidRegexp2() throws Exception {
```

Comment:
It is almost the same as `testHeaderWithInvalidRegexp()` with the difference in the input file. If you keep existing test and just update its input and expected exception message, would that be enough, so we don't need `testHeaderWithInvalidRegexp2()`?

Reply:
Yes, let's not be blocked by semaphore.